### PR TITLE
Fix translation issues on PDP and RTL placement for Tooltips

### LINKED ISSUE #2287: "Add Option" tooltip is always out of view
### Expected behavior
Tooltip should be in the viewport

### Actual Behavior
It never is
dumb_product_and_cart_completed

### Steps to Reproduce the Behavior
1. Create a product
1. Mouse over the "plus" icon
1. Observe tooltip is partially out of view

### Versions
```
Node: 7.10.0
NPM: 4.2.0
Meteor Node: 4.8.2
Meteor NPM: 4.5.0
Reaction CLI: 0.8.2
Reaction: 1.2.0
Reaction branch: release-1.2.0
Docker: 17.03.1-ce
```

### LINKED ISSUE #2641: PDP / Dashboard components i18n
As reported in #2623

Take a look at "Available Options" on the PDP -> you'll see the field only translates on refresh after i18n update.

basic_reaction_product_and_2__reaction_reset_-y____reaction__node_
basic_reaction_product_and_2__reaction_reset_-y____reaction__node_

### PR DESCRIPTION
**Pull request by kieckhafer**
Resolves #2641 
Resolves #2287 

**_Divider_**
The divider component is registering as `pure`. 

As of right now, it cannot be a pure component, as the `i18nKey` never changes, even with a language switch, it's still an `i18nKey` like `app.public`. Since the key stays the same, the props don't change, the component doesn't update.

**_Tooltips_**
Tooltips were overflowing outside the window area. 

I changed the tether to use `window` instead of the current element, and that seems to have fixed it everywhere.

**_Remaining Translations_**
[WIP] please help i don't know.

### CODE REVIEW

**Comment by kieckhafer** [comment]
-Tooltips are fixed.
- Translations on PDP are fixed.
- Still having issues with a few of the panels in the dashboard, and I am struggling to figure it out, because it seems very random which ones don't work. Some are a new React panel title, some are older Blaze templates, but they all have available translations, and other React / Blaze templates with the exact same setup are working.

These three are the remaining culprits, if anyone has any ideas... help please!

reaction
reaction 2
x_revolve_stanton_top

**Comment by zenweasel** [comment]
@kieckhafer Is this ready to go? Should we just merge the fixes you have in and open issues for the few remaining?

**Comment by kieckhafer** [comment]
The above issue are still there, but I think we should push and make a new ticket. AFAIK all translations except these three will be fixed with this update.

**Comment by kieckhafer** [comment]
Here's a ticket for the rest: https://github.com/reactioncommerce/reaction/issues/2672

**Comment by zenweasel** [comment]
@kieckhafer Do you want to assign reviewers?

**Comment by zenweasel** [comment]
Ok, I guess I deserved that

**Comment by zenweasel** [comment]
It looks like "Dashboard" is still untranslated. Or that could be just a missing translation? Is there a certain language I should test with.

cart_checkout

**Comment by aaronjudd** [comment]
You'll need to run lingohub and get all the translations for the new coreTitle..

**Comment by zenweasel** [comment]
How do I create translations from a branch again?

**Comment by aaronjudd** [comment]
repository_setting___lingohub
 But I see you just figured it out.. but you can create the PR now...

**Comment by zenweasel** [comment]
"Dashboard" still untranslated

**Comment by zenweasel** [comment]
"Dashboard" is now translated.

**Comment by zenweasel** [comment]
Looks like all the translations in the variant panel are missing

basic_reaction_product

Look to be translated in `master`.

basic_reaction_product

**Comment by zenweasel** [comment]
I'm not sure what the best course of action here, to keep testing this or just merge and open new issues. I think this fixes the specific issues pointed out in ticket?

File: imports/plugins/core/accounts/server/i18n/ar.json [hunk 1]
```diff
@@ -10,13 +10,19 @@
...
+          "changeShopOwnerWarn": "أنت على وشك تغيير مالك المحل. سيتم استبدال المالك الحالي",
+          "changeMktOwnerWarn": "أنت على وشك تغيير مالك ماركتبليس. سيتم استبدال المالك الحالي",
           "accountsSettingsLabel": "الحسابات",
           "accountsSettingsTitle": "الحسابات",
           "accountsSettingsDescription": "إعدادت الحساب"
         },
         "groupsInvite": {
           "name": "جون سميث",
-          "email": "johnsmith@reactioncommerce.com"
+          "email": "johnsmith@reactioncommerce.com",
```
Comment (kieckhafer) [inline imports/plugins/core/accounts/server/i18n/ar.json]: Should we change this to a non-reaction email?

File: imports/plugins/core/accounts/server/i18n/bg.json [hunk 2]
```diff
@@ -10,13 +10,19 @@
...
+          "changeShopOwnerWarn": "На път сте да промените собственика на магазина. Сегашният собственик ще бъде заменен",
+          "changeMktOwnerWarn": "На път сте да промените собственика на Marketplace. Сегашният собственик ще бъде заменен",
           "accountsSettingsLabel": "Профили",
           "accountsSettingsTitle": "Профили",
           "accountsSettingsDescription": "Настройки на профила"
         },
         "groupsInvite": {
           "name": "Джон Смит",
-          "email": "johnsmith@reactioncommerce.com"
+          "email": "johnsmith@reactioncommerce.com",
```
Comment (kieckhafer) [inline imports/plugins/core/accounts/server/i18n/bg.json]: Should we change this to a non-reaction email?

Comment (kieckhafer) [inline imports/plugins/core/accounts/server/i18n/bg.json]: Should we change this to a non-reaction email on the `en` version??

Comment (aaronjudd) [inline imports/plugins/core/accounts/server/i18n/bg.json]: yes should be `email@localhost` -> and should not be translated in LingoHub (save as a phrase).

**Pull request merged**